.vscode/script.js

```diff
@@ -0,0 +1,32 @@
+let firstName = document.getElementById("firstName")
+let lastName = document.getElementById("lastName")
+let age = document.getElementById("age")
+let phone = document.getElementById("phone")
+let email = document.getElementById("email")
+let male = document.getElementById("male")
+let female = document.getElementById("female")
+let error = document.getElementsByTagName("p")[0]
```

Comment:
@prateeksanodiya0209  Always try to give a class or id to the tag and get document by it in place of getting by `tag` for a best practice.

this comment is for the `<p>` tag